Makefile

```diff
@@ -5,7 +5,7 @@ start-services:
 	docker-compose up -d nginx postgres redis elasticsearch
 
 start-clay:
-	cd app && npm i && npm run start:dev
+	cd app && cp .env.sample .env && npm i && npm run start:dev
```

Comment:
This will overwrite the `.env` file every time you start the app.

Reply:
Nice catch! Do you think it would be best to add this step in the README?

Reply:
Yes, that sounds like a good idea